ops/framework.py

```diff
@@ -484,6 +484,14 @@ def define_event(cls, event_kind: str, event_type: 'Type[EventBase]'):
 
         event_type: a type of the event to define.
 
+        ops uses a labeling system to track and reconstruct events between hook
```

Comment:
This feels a bit out of the blue after the parameter spec above.
I would add some intro sugar such as:

Attempting to define the the same event kind more than once will raise <exception type>. This is because...